app/javascript/crayons/MentionAutocompleteTextArea/MentionAutocompleteTextArea.jsx

```diff
@@ -15,30 +15,49 @@ const MIN_SEARCH_CHARACTERS = 2;
 const MAX_RESULTS_DISPLAYED = 6;
 
 /**
- * Helper function to copy all styles and attributes from the original textarea to new autocomplete textarea before removing the original node
+ * Helper function to copy all styles and attributes from the original textarea to new autocomplete textareas before removing the original node
  *
- * @param {element} originalNodeToReplace The DOM element that should be replaced
- * @param {element} newNode The DOM element that will receive all attributes and styles of the original node.
+ * @param {object} options
+ * @param {element} options.originalNodeToReplace The DOM element that should be replaced
+ * @param {element} options.plainTextArea The DOM element to be used in the 'non-autosuggest' state. It will receive all attributes and styles of the original node.
+ * @param {element} options.comboboxTextArea The DOM element to be used in the 'autosuggest' state. It will receive all attributes and styles of the original node.
  */
-const replaceTextArea = (originalNodeToReplace, newNode) => {
-  // Make sure all attributes are copied to the autocomplete textarea
-  const attributes = originalNodeToReplace.attributes;
-  Object.keys(attributes).forEach((attributeKey) => {
-    newNode.setAttribute(
-      attributes[attributeKey].name,
-      attributes[attributeKey].value,
-    );
-  });
+const replaceTextArea = ({
+  originalNodeToReplace,
+  plainTextArea,
+  comboboxTextArea,
+}) => {
+  const newNodes = [plainTextArea, comboboxTextArea];
 
-  // Make sure all styles are copied to the autocomplete textarea
-  newNode.style.cssText = document.defaultView.getComputedStyle(
+  const attributes = originalNodeToReplace.attributes;
```

Comment:
Refactor (non-blocking): Prefer destructuring. I know you mentioned adding the eslint rule, so we should be good in future PRs. 😎